manifests/params.pp

```diff
@@ -78,6 +78,7 @@
   # OpenNebula KVM driver parameters
   $kvm_driver_emulator       = hiera ('one::oned::kvm_driver_emulator', 'undef')
   $kvm_driver_nic_attrs      = hiera ('one::oned::kvm_driver_nic_attrs', 'undef')
+  $kvm_driver_raw_attrs      = hiera ('one::oned::kvm_driver_raw_attrs', '[ DATA="<cpu mode=\'host-model\'/>", TYPE="kvm" ]')
```

Comment:
I'd prefer an empty default value over a defined setting. People might have diverse clusters which would render this setting at least problematic.